website/templates/include/profile/jobs.mako

```diff
@@ -112,7 +112,8 @@
                 <button
                         type="submit"
                         class="btn btn-success"
-                    >Save</button>
+                        data-bind="text: savingText(), enable : !saving()">
```

Comment:
I believe you can bind on both `enable` and `disable`. Binding to `disable` would avoid negating the result of `saving()`, making it slightly more readable.

This is entirely subjective, so as a wise man once told me in a code review - follow your heart :heart:.